Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -155,6 +155,8 @@ function xASL_wrp_Quantify(x, PWI_Path, pathOutputCBF, M0Path, SliceGradientPath
 
 %% ------------------------------------------------------------------------------------------------
 %% 2.   Prepare M0
+M0_parms = xASL_adm_LoadParms(x.P.Path_M0_parms_mat, x);
```

Comment:
I moved this here because it was incorrect to have it only in the single branch of IF - so I moved it before this IF so that this is run in all branches because M0_parms were also used even for ApplyQuantification(5) == 0.

On the other hand, M0_parms is only used later in the code, so I moved it to the place where it is really needed.

Other than that - it won't crash with a missing file. It will report a warning and provide an empty structure.